src/translations/en/privacy.json

```
@@ -1,17 +1,275 @@
+        "bullet_point0": "Pending the data controller rectifying inaccurate or incomplete personal data",
+        "bullet_point1": "If you have submitted an objection to the processing (see below for more details)",
+        "bullet_point2": "If the personal data are necessary to establish or defend a legal claim"
+      },
+      "paragraph1": "If restricted processing is granted, the data controller will notify you before the restriction is lifted."
+    },
+    "section3": {
+      "heading": "Right to erasure",
+      "paragraph0": "You have the right to demand that we erase personal data about you. If you would like your personal data to be erased, please contact the data controller. It is important that your request states why you want your personal data erased and, if possible, what personal data you wish to be erased.",
+      "paragraph1": "Please note that legislation provides exemptions from the right to erasure in some cases. This could be cases where we process personal data to fulfil a statutory duty or to address important social interests, such as archiving, research and statistics. In cases where the users themselves have registered content in the service, we are not able to erase user information without also erasing the registered content.",
+      "paragraph2": "Please note that there is also a limited possibility of erasing personal data that are distributed via open interfaces."
+    },
+    "section4": {
+      "heading": "Right to object",
+      "paragraph0": {
+        "intro": "You have a right to object to processing of your personal data on certain conditions defined in the <0>GDPR Article 21</0>. This applies if:",
+        "bullet_point0": "Legal grounds for the processing of personal data are based on legitimate interests, reasons of public interest or when exercising public authority",
+        "bullet_point1": "Processing of personal data entails direct marketing or profiling",
+        "bullet_point2": "Personal data are processed for scientific or historical research purposes or statistical purposes"
+      },
+      "paragraph1": "The service does not generally satisfy these conditions, with the exception of the processing of author information and in cases where personal data are used for statistical purposes. When personal data are used for statistical purposes, the data are anonymised.",
+      "paragraph2": {
+        "intro": "However, please note the following:",
+        "bullet_point0": "If there is a special need to stop the processing, for example if you have a need of protection, or a confidential address or similar, please contact the data controller",
```

```suggestion
        "bullet_point0": "If there is a special need to stop the processing, for example if you have a need for protection, or a confidential address or similar, please contact the data controller",
```
(in need of / a need for)